### DISCUSSION TITLE
如何解决有些游戏无法下载的问题

### DISCUSSION THREAD

**Discussion by guizhenyu0512** [Gamuxorg/bbs#345] [游戏或网站运行反馈]
您好，在linuxgame.cn上的游戏列表中有些游戏找不到下载方法，比如

https://www.linuxgame.cn/%E8%8B%B1%E9%9B%84%E6%97%A0%E6%95%8C3%E6%AD%BB%E4%BA%A1%E9%98%B4%E5%BD%B1vcmi%E7%89%88#2
https://www.linuxgame.cn/hollowknight

我该如何下载这类游戏呢？ 访问密码又是什么？

**Comment by shouhuanxiaoji** [comment]
因为下载链接以前是http，网站启用了https可能导致出了问题，后台把下载链接改成https就好了。
暂时还不清楚有多少页面出了影响。。
如果你有兴趣我可以给你一个编辑权限，帮助修改一下

**Comment by guizhenyu0512** [comment]
好的，我愿意帮忙更改，但是我不太清楚如何使用github，我应该在哪更改，更改哪些文件呢？

**Comment by shouhuanxiaoji** [comment]
你在gamux登陆过吗？登陆过的话，我可以给你添加一下编辑权限，就可以编辑了

**Comment by guizhenyu0512** [comment]
是的，我登录过，就用我这个帐号通过github给予权限登录的。

**Comment by shouhuanxiaoji** [comment]
好了

**Comment by guizhenyu0512** [comment]
抱歉，不用了，我找到编辑方法了。

----------------------------------------------

谢谢， 我在一些游戏中看到编辑的按钮了，可是有些游戏我没有看到按钮在哪， 比如：
https://www.linuxgame.cn/beholder#1
https://www.linuxgame.cn/super-meat-boy

viewsource 也没有找到编辑的链接在哪。不知道从你那边看有没有这种情况？

**Comment by shouhuanxiaoji** [comment]
可以加群274328087讨论一下问题
